packages/release-mastermind/src/contexts/issues.ts

```diff
@@ -8,58 +1,154 @@
+/** @format */
+
+import * as core from "@actions/core"
+import { Context } from "@actions/github/lib/context"
+import { LoggingDataClass, LoggingLevels } from "@videndum/utilities"
+import { log } from ".."
+import { Config, Runners, SharedConfig } from "../action"
+import { CurContext, IssueContext, Version } from "../conditions"
+import { Utils } from "../utils"
+import { Contexts } from "./methods"
+import { AssignProject } from "./methods/assignProject"
+import { CreateBranch } from "./methods/createBranch"
+
+/**
+ * The issue configuration
+ */
+export interface IssueConfig extends SharedConfig {
+	/**
+	 * Assign project configuration.
+	 */
+	assignProject?: AssignProject[]
+	/**
+	 * Open branch configuration
+	 */
+	createBranch?: { [label: string]: CreateBranch }
+}
+
+/**
+ * The issue class.
+ * @private
+ */
 export class Issues extends Contexts {
-  context: IssueContext
-  config: IssueConfig
-  constructor(
-    util: Utils,
-    runners: Runners,
-    configs: Config,
-    curContext: CurContext,
-    dryRun: boolean
-  ) {
-    if (curContext.type !== 'issue')
-      throw new LoggingDataClass(LoggingLevels.error, 'Cannot construct without issue context')
-    super(util, runners, configs, curContext, dryRun)
-    this.context = curContext.context
-    if (!configs.issue)
-      throw new LoggingDataClass(LoggingLevels.error, 'Cannot start without config')
-    this.config = configs.issue
-  }
+	context: IssueContext
+	config: IssueConfig
+	constructor(
+		util: Utils,
+		runners: Runners,
+		configs: Config,
+		curContext: CurContext,
+		dryRun: boolean
+	) {
+		if (curContext.type !== "issue")
+			throw new LoggingDataClass(
+				LoggingLevels.error,
+				"Cannot construct without issue context"
+			)
+		super(util, runners, configs, curContext, dryRun)
+		this.context = curContext.context
+		if (!configs.issue)
+			throw new LoggingDataClass(
+				LoggingLevels.error,
+				"Cannot start without config"
+			)
+		this.config = configs.issue
+	}
+
+	/**
+	 * Parse the Issue Context
+	 * @author IvanFon, TGTGamer, jbinda
+	 * @since 1.0.0
+	 */
+	static async parse(
+		utils: Utils,
+		config: Config,
+		context: Context
+	): Promise<IssueContext | undefined> {
+		const issue = context.payload.issue
+		if (!issue) {
+			return
+		}
+
+		log(
+			LoggingLevels.debug,
+			`context.payload.issue: ` + JSON.stringify(context.payload.issue)
+		)
+
+		const labels = await utils.parsingData.labels(issue.labels).catch((err) => {
+			log(LoggingLevels.error, `Error thrown while parsing labels: `, err)
+			throw err
+		})
+
+		const currentVersion: Version = await utils.versioning
+			.parse(config, config.issue?.ref)
+			.catch((err) => {
+				log(LoggingLevels.error, `Error thrown while parsing versioning: `, err)
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Videndum/action-masterminds/pullRequest?prid=7872005)
```suggestion
        log(LoggingLevels.error, "Error thrown while parsing versioning: ", err)
```